packages/moonstone/VideoPlayer/MediaSliderDecorator.js

```diff
@@ -198,6 +207,7 @@ const MediaSliderDecorator = hoc((config, Wrapped) => {	// eslint-disable-line n
 					onMouseOver={this.handleMouseOver}
 					onMouseOut={this.handleMouseOut}
 					onMouseMove={this.handleMouseMove}
+					onTouchMove={platform.touch ? this.handleTouchMove : void 0}
```

Comment:
I'm not sure this ternary is really buying us much particularly for how limited VP is used. I think I'd slightly prefer either always attaching the handler (which would require always binding it at construction) or adding an `if` block and appending it `rest`. That said, not a blocker so just for consideration.